src/Message/MessageBuilder.php

```diff
@@ -85,8 +85,8 @@ private function getFullName(array $params): string
 
     /**
      *     @var string
-     *     @var string $first
-     *     @var string $last
+     *     @var string
+     *     @var string
```

Comment:
These annotations are for

```php
parseAddress(string $address, array $variables): string
```

So they don't seem to be consistent. Can you please address this?

Reply:
oh yes. This should be: 
```
    /**
     * @param array $variables {
     *
     *     @var string $full_name
     *     @var string $first
     *     @var string $last
     * }
     */
```